cszczepaniak/cribbly-backend
# add delete teams endpoint

# What I added (link any issues addressed)
Closes #29 
# How to test it

- Ensure there are teams in the database
- Call ```DELETE https://localhost:5001/api/team/delete```
- Include the following as the body

```
{
    "id": int
}
```

- See that it deletes the record and sets the players to not having a team
